features/step_definitions/collection_steps.rb

```diff
@@ -115,6 +115,21 @@
   step %{I should see "Updated #{name}"}
 end
 
+Given /^I have added (?:a|the) co\-owner "([^\"]*)" to collection "([^\"]*)"$/ do |name, title|
```

Comment:
`Given "I have added (a)(the) co-owner {string} to collection {string}" do |name, title|` will probably work here and make Reviewdog happy.

(If you want to get really ambitious, you could check if the test suite still passes when making the same change to the co-moderator step definition this is based on. That's totally optional, though.)